server/src/repositories/Training/OptionRepository.ts

```diff
@@ -20,6 +20,17 @@ export async function getOptionsByModuleItem(
   return OptionMap.optionsToDomain(knexResult);
 }
 
+export async function getCorrectOptionsWithModuleItemByModule(
```

Comment:
I think this name could be improved. My first thought is `getCorrectOptionsForEachModuleItem`, but that's just as long. Conceptually, we're getting all the correct answers. If this was a pen & paper test, that would be called the answer sheet. `getAnswerSheet`?